app/models/alchemy/site.rb

```diff
@@ -70,21 +70,18 @@ def find_in_aliases(host)
       end
     end
 
-    before_create do
-      # If no languages are present, create a default language based
-      # on the host app's Alchemy configuration.
-
-      if languages.empty?
-        default_language = Alchemy::Config.get(:default_language)
-        languages.build(
-          name:           default_language['name'],
-          language_code:  default_language['code'],
-          frontpage_name: default_language['frontpage_name'],
-          page_layout:    default_language['page_layout'],
-          public:         true,
-          default:        true
-        )
-      end
+    # If no languages are present, create a default language based
+    # on the host app's Alchemy configuration.
+    before_create unless: 'languages.any?' do
```

Comment:
Would it not make sense to refactor this into a private method `create_default_language`?